### PR TITLE
fix redirection issues for evaluation results

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Evaluation Result - Redirection Issues#425](https://github.com/nerubia/kh360-react/issues/425)

### CODE REVIEW

File: src/features/evaluation-results/[id]/view-evaluation-results-footer.tsx [hunk 1]
```diff
@@ -1,12 +1,18 @@
 import { useNavigate } from "react-router-dom"
 import { Button } from "../../../components/ui/button/button"
 import { Icon } from "../../../components/ui/icon/icon"
+import { useAppSelector } from "../../../hooks/useAppSelector"
 
 export const ViewEvaluationResultsFooter = () => {
   const navigate = useNavigate()
+  const { previousUrl } = useAppSelector((state) => state.evaluationAdministrations)
```
Comment (jatnerubia) [inline src/features/evaluation-results/[id]/view-evaluation-results-footer.tsx:8]: This is for evaluation administrations only. Should we move this to the app slice?

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

**Comment by emmaacha** [comment]
Done ^^

**Review by chadiegilnerubia — APPROVED** [review]
:1st_place_medal:

**Review by jatnerubia — APPROVED** [review]
Nice

**Pull request merged**